eth/vm/forks/homestead/__init__.py

```diff
@@ -18,7 +17,13 @@
 
 class MetaHomesteadVM(FrontierVM):
     support_dao_fork = True
-    dao_fork_block_number = DAO_FORK_MAINNET_BLOCK
+    dao_fork_block_number = None  # type: Optional[BlockNumber]
```

Comment:
Maybe move to private `_dao_fork`...?

- Helps make sure that there aren't any stray references to `VM.dao_fork_block_number` out there
- Encourages new usages to call the `get_dao_fork`... because the other one looks private.